code/modules/mob/living/carbon/human/species/working_joe/_species.dm

```diff
@@ -18,14 +18,19 @@
 	name_plural = "Hazard Joes"
 	emote_panel_type = /datum/joe_emote_panel/hazard
 
+/datum/species/synthetic/colonial/working_joe/hazard/handle_death(mob/living/carbon/human/dying_joe, gibbed)
+	if(!gibbed) //A gibbed Joe won't have a death rattle
+		playsound(dying_joe.loc, "hj_death", 25, FALSE)
+	return ..()
```

Comment:
this will play it twice, you don't need to call parent here though. the parent hndle_death proc doesn't do anything